src/Dont/DeSerialize.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Dont;
+
+use Dont\Exception\NonDeSerializableObject;
+use Dont\Exception\TypeError;
+
+trait DeSerialize
```

Comment:
Acceptable to me in selfish preference order: `Deserialise` `Deserialize` `Unserialise` `Unserialize`  (note, no capital `S` in any)